doc/develop/test/blackbox.rst

```diff
@@ -0,0 +1,120 @@
+.. _blackbox:
```

Comment:
Please move this into a subfolder and make twister specific. _blackbox is very generic of a reference if this is only about twister.